<launch>
	<include file="$(find location_monitor)/launch/my_turtlebot_world.launch" />
	

  	<!-- Rviz base -->
 	<include file="$(find turtlebot_rviz_launchers)/launch/view_robot.launch"/>
	
	
	<node name="Case1" pkg="location_monitor" type="Case1.py" output="screen">
		<param name="vel_out_topic" value="/cmd_vel_mux/input/teleop" />		
    		
		<!-- to remap the speed subscribed from other package -->
		<remap from="vel_in" to="cmd_vel"/>
	
	</node>

	
</launch>
